src/ParallelAlgorithms/ApparentHorizonFinder/Callbacks/ObserveFieldsOnHorizon.hpp

```diff
@@ -0,0 +1,156 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
```

Comment:
Since the tests are in a nonstandard location, put a comment somewhere (probably near the top of each file) saying where they are.